config/routes.rb

```diff
@@ -55,6 +55,8 @@
 
   get 'search' => 'search#index'
   post 'create_multiple_evidences' => 'evidence#create_multiple'
+  get 'trash' => 'revisions#trash'
```

Comment:
At the end of the day we're acting on the Revision resource, so I think that works, maybe even use "/revisions/deleted" which is more meaningful, although I could be persuaded to keep trash as it's familiar enough.